mcu-util/src/messaging/can/canfd.rs

```diff
@@ -201,26 +201,20 @@ impl MessagingInterface for CanRawMessaging {
         };
 
         if let Some(bytes) = bytes {
-            let mut buf: [u8; 64] = [0u8; 64];
+            let mut buf: [u8; CANFD_DATA_LEN] = [0u8; CANFD_DATA_LEN];
```

Comment:
```suggestion
            let mut buf = [0u8; CANFD_DATA_LEN];
```
No need to annotate, the type is already clear.